tests/foreman/endtoend/utils.py

```diff
@@ -25,15 +26,15 @@ def client_provisioning(
         """
         if not setting_is_set('clients'):
             return
-        with VirtualMachine(distro=DISTRO_RHEL6) as vm:
+        with VMBroker(nick='rhel6', host_classes={'host': ContentHost}) as host:
```

Comment:
You should be able to use the rhel6 fixture provided by pytest_fixtures/broker.py in this test.

Reply:
I believe that is true is if the changes are being made to a test. However, this is not a test, it's a class method with some default values that is being used by a test. Adding `rhel6_contenthost` is giving the error `SyntaxError: non-default argument follows default argument`

Reply:
Good point, I didn't see the context so assumed it was a test!